Misc/NEWS.d/next/Library/2022-07-03-23-09-45.gh-issue-94512.9wtOYT.rst

```diff
@@ -0,0 +1,3 @@
+:meth:`resource.prlimit` changed its signature from
+``prlimit(pid, resource, [limits], /)`` to
+``prlimit($module, pid, resource, limits=None, /)``.
```

Comment:
Doesn't require a NEWS entry for this, as the signature hasn't _really_ changed (`None` wasn't valid for limits before and its behaviour now is sensible)

Reply:
Addressed.